pkgs/servers/monitoring/telegraf/default.nix

```diff
@@ -15,10 +15,11 @@ buildGoModule rec {
     sha256 = "045wjpq29dr0s48ns3a4p8pw1j0ssfcw6m91iim4pkrppj7bm2di";
   };
 
-  runVend = true;
-  vendorSha256 = "0c2sayg49b2rq3fnrbf741b6zy8byhwxlnxkhf5160gzqn6jy2rw";
+  patches = [
+    ./gomod.patch
```

Comment:
Could you use fetchpatch here?

https://github.com/influxdata/telegraf/commit/a0d0c80c373dd1896d449e9f3534a157d78fc866.patch

This is the PR:

https://github.com/influxdata/telegraf/pull/7988

Reply:
```
vendor/github.com/shirou/gopsutil/host/host_bsd.go:25:13: undefined: unix.SysctlTimeval
``

Looks like it works with 2.20.6 and breaks with a different issue on 2.207.

Reply:
Ok. I will have a look: https://github.com/shirou/gopsutil/issues/591

Reply:
Bumped to 2.20.6: Let's see if that makes CI happy: https://github.com/influxdata/telegraf/commit/63e1f41d8ff246d191d008ff7f69d69cc34b4fae.patch

Reply:
Builds with patch, generates the same `vendorSha256`.

Reply:
Looks this patch breaks FreeBSD now.... https://app.circleci.com/pipelines/github/influxdata/telegraf/1327/workflows/6b7efed4-74fb-45bf-a3c7-2978f9bdf5bb/jobs/62496

However I think this should be good enough for nixpkgs.